libkbfs/md_util.go

```diff
@@ -267,12 +267,19 @@ func getMergedMDUpdates(ctx context.Context, config Config, id tlf.ID,
 // and unmerged branch, between the merge point for that branch and
 // startRev (inclusive).  The returned MDs are the same instances that
 // are stored in the MD cache, so they should be modified with care.
+// If bid is NullBranchID, it returns an empty MD list.
```

Comment:
I'd argue that the mdserver code should also return an error, but I don't feel too strongly about it.